include/unix/osd.h

```diff
@@ -37,6 +37,8 @@
 #include <unistd.h>
 #include <errno.h>
 
+#define FI_WAIT_MUTEX_CRITSEC FI_WAIT_MUTEX_COND
```

Comment:
But none of the sites where the new macro is used **require** the new macro nor an `#ifdef`.  Most of them just require you to add an additional `case FI_WAIT_CRITSEC_COND` line.  From my quick skim of the code, it looks like you would also need to update two `if` cases with simple logic modifications.